src/main/resources/application.conf

```diff
@@ -12,4 +12,5 @@ centaur {
   # case will be in a subdirectory named FOO with files FOO.wdl, FOO.inputs, and FOO.options. This is not
   # particularly robust at the moment.
   standardTestCasePath: "src/main/resources/standardTestCases"
+  # optionalTestPath: "/path/to/test/directory"
```

Comment:
Since @geoffjentry wants to match the README: `optionalTestPath=/some/path/to/tests`